.github/workflows/ci-badger.yaml

```diff
@@ -31,7 +34,17 @@ jobs:
         go-version: 1.22.x
 
     - name: Run Badger storage integration tests
-      run: make badger-storage-integration-test
+      run: |
+        case ${{ matrix.version }} in
+          v1)
+            SPAN_STORAGE_TYPE=badger \
```

Comment:
This wasn't passed before, is it needed?

Reply:
No, removed it.